google/cloud/bigtable/row_data.py

```diff
@@ -487,6 +491,9 @@ def __iter__(self):
                 if self.state != self.NEW_ROW:
                     raise ValueError("The row remains partial / is not committed.")
                 break
+            except InvalidRetryRequest:
+                self._cancelled = True
```

Comment:
Will this show up as cancelled status instead of success?

Reply:
no it should end in a success...it will just break out of the loop